### DISCUSSION TITLE
Is it possible to have a granular date display (X axis) on email graphs in Postal

### DISCUSSION THREAD

**Discussion by helmuc** [postalserver/postal#3207] [Help with using Postal]
Dear Postal Support Team,

We are using Postal as our SMTP server and would like to gain more detailed insights into our email sending patterns. Currently, on the X-axis of the email graph, the dates are displayed in broad intervals, which makes it challenging to pinpoint the exact days on which emails were sent out.

Could you please advise if there is a way to increase the granularity of the X-axis to show individual dates, or if there are any settings that would allow us to view the email activity on a day-by-day basis? We have attached a screenshot for reference.

Thank you for your assistance!

Best wishes,
Helmuts

![chrome_uhVIu9eNc8](https://github.com/user-attachments/assets/d37bd2b3-7774-4e4f-888c-68e3ff2dee0d)

**Comment by willpower232** [comment]
Its not currently a feature but I believe if you look in the database, you should be able to see where the numbers come from.

I think there is a whole table devoted to them so with the right frontend plugin, I bet it could be achieved.
